Assets/ScreenManager/Editor/ScreenManagerEditor.cs

```diff
@@ -289,9 +291,18 @@ public override void OnInspectorGUI()
             GUILayout.BeginHorizontal(EditorStyles.toolbar);
             {
                 GUILayout.BeginHorizontal(GUI.skin.FindStyle("Toolbar"));
-                //GUILayout.FlexibleSpace();
-                searchString = GUILayout.TextField(searchString, GUI.skin.FindStyle("ToolbarSeachTextField")/*, GUILayout.MaxWidth(300)*/);
-                if (GUILayout.Button("", GUI.skin.FindStyle("ToolbarSeachCancelButton")))
+                
+                // Below is a result of Unity's typo in versions prior to 2021.3.28
+                toolbarSearchTextFieldStyle = GUI.skin.FindStyle("ToolbarSearchTextField");
+                if (toolbarSearchTextFieldStyle == null)
+                    toolbarSearchTextFieldStyle = GUI.skin.FindStyle("ToolbarSeachTextField");
+
+                toolbarSearchCancelButtonStyle = GUI.skin.FindStyle("ToolbarSearchCancelButton");
```

Comment:
```suggestion
                if (toolbarSearchTextFieldStyle == null)
                        toolbarSearchCancelButtonStyle = GUI.skin.FindStyle("ToolbarSearchCancelButton");
```

Reply:
..could just do another null check like this, if you were thinking about some optimisation?

Reply:
Yeah, that sounds even better, just lookup if there is no cached style. Does `GUI.skin` change at runtime though?

Reply:
No it doesn't. Would be better to put in Awake equivalent for Editor component if you know of a method?

Reply:
Seems like [messages](https://docs.unity3d.com/ScriptReference/Editor.html) list `Awake` and `OnEnable`.

Reply:
I've moved to OnEnable now...